README.adoc

```diff
@@ -19,3 +19,34 @@ image:https://img.shields.io/badge/Google%20Group-Gatling-blue?style=for-the-bad
 [IMPORTANT]
 
 The documentation now lives on the Gatling website and can be found https://gatling.io/docs/current/extensions/gradle_plugin/[here].
+
+== Dev testing
+
+To publish the plugin to your local Maven repository, add the following to the `+build.gradle+` file:
+
+....
+apply plugin: 'maven-publish'
+group 'io.gatling'
+version '0.0.1-SNAPSHOT'
+publishing {
+  repositories {
+    mavenLocal()
+  }
+}
+....
+
+And then publish with the command `+gradlew publishToMavenLocal+`.
+
+In the Gradle project where you want to use the locally published plugin, add a `+settings.gradle+` file containing the following:
+
+....
+pluginManagement {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    gradlePluginPortal()
+  }
+}
+....
+
+And then configure the use of the `io.gatling.gradle` plugin as normal, with the same version number used to publish locally.
```

Comment:
The gradle documentation explain how to manually test a plugin:
https://docs.gradle.org/current/userguide/testing_gradle_plugins.html#manual-tests

maven deployment is not the preferred way for gradle and we don't need to pollute our environment

More, we want to ensure that our current code is what we are currently testing.